src/pages/settings/Security/SecuritySettingsPage.tsx

```diff
@@ -53,13 +33,13 @@ function SecuritySettingsPage(props) {
             },
         ];
 
-        return _.map(baseMenuItems, (item) => ({
+        return baseMenuItems.map((item) => ({
             key: item.translationKey,
-            title: translate(item.translationKey),
+            title: translate(item.translationKey as TranslationPaths),
```

Comment:
Can you create a follow up PR to fix this assertion?

here is a code snippet that should work @tienifr:
```ts
    const menuItems: MenuItemWithLink[] = useMemo(() => {
        const baseMenuItems = [
            {
                translationKey: 'twoFactorAuth.headerTitle',
                icon: Expensicons.Shield,
                action: waitForNavigate(() => Navigation.navigate(ROUTES.SETTINGS_2FA.getRoute())),
            },
            {
                translationKey: 'closeAccountPage.closeAccount',
                icon: Expensicons.ClosedSign,
                action: waitForNavigate(() => Navigation.navigate(ROUTES.SETTINGS_CLOSE)),
            },
        ] as const;

        return baseMenuItems.map((item) => ({
            key: item.translationKey,
            title: translate(item.translationKey),
            icon: item.icon,
            onPress: item.action,
            shouldShowRightIcon: true,
            link: '',
        }));
    }, [translate, waitForNavigate]);
```